http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala

```diff
@@ -140,17 +140,19 @@ class ConfigReaderSpec extends AnyFlatSpec with Matchers {
         WorkflowsAppConfig(
           ChartName("/leonardo/workflows-app"),
           ChartVersion("0.24.0"),
-          ReleaseNameSuffix("workflows-app-rls"),
-          NamespaceNameSuffix("workflows-app-ns"),
-          KsaName("workflows-app-ksa"),
+          ReleaseNameSuffix("wfa-rls"),
```

Comment:
I assume this is because we don't like the `-` in the name? Can we preserve a human readable name like `workflows` though?

Reply:
This was actually because this led to super long names like `workflows-app-mkcjzg-workflows-app-rls-cbas-depl-78ccb5fd4d9r7v`. Open to other ideas. I think Saloni is planning on shortening hers to cra.

Reply:
Yeah I am open to suggestions too for Cromwell Runner app since having `cromwell-runner-app` appear twice in names for making it go over 63 character limit and throwing an error.